Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -1047,8 +1047,16 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
           position: position,
           .consecutiveDeclarationsOnSameLine,
           fixIts: [
-            FixIt(message: .insertSemicolon, changes: .makePresent(semicolon))
-          ],
+            node.lastToken(viewMode: .sourceAccurate).map {
+              FixIt(
+                message: .insertNewline,
+                changes: [
+                    .replaceTrailingTrivia(token: $0, newTrivia: $0.trailingTrivia + .newlines(1))
```

Comment:
```suggestion
                  .replaceTrailingTrivia(token: $0, newTrivia: $0.trailingTrivia + .newlines(1))
```

Didn’t swift-format catch this?

Reply:
Forgot to run it🫠